docs/RFCS/explain_debug.md

```diff
@@ -0,0 +1,119 @@
+- Feature Name: explain_debug
+- Status: in-progress
+- Start Date: 2015-08-10
+- RFC PR: #2047
+- Cockroach Issue:
+
+# Summary
+
+Add support for an `EXPLAIN DEBUG` statement. The `EXPLAIN DEBUG` statement
+will cause simple (single table or index) select statements to display the
```

Comment:
The difficulty with multi-table or multi-index queries is that the mapping from key accessed to row is going to be quite a bit more complex. I'll keep that in mind while implementing, though. I'd like to make this fully general purpose, but I don't have a good sense for how to do so yet.